doc/examples/segmentation/plot_regionprops_table.py

```diff
@@ -0,0 +1,106 @@
+"""
+===================================================
+Explore and visualize region properties with pandas
+===================================================
+
+This toy example shows how to compute the size of every labelled region in a
+series of 10 images. We use 2D images and then 3D images. The blob-like
+regions are generated synthetically. As the volume fraction (i.e., number of
+pixels or voxels covered by the blobs) increases, the number of blobs
+(regions) decreases, and the size (area or volume) of a single region can get
```

Comment:
```suggestion
(regions) decreases, and the size (area or volume) of a single region gets
```

Reply:
I would actually keep the probabilistic touch.